scripts/jenkins/qa_openstack.sh

```diff
@@ -361,8 +367,11 @@ outputs:
   server_floating_ip:
     value: { get_attr: [ my_floating_ip, floating_ip_address ] }
 EOF
-
-heat stack-create -f $(readlink -e $PWD/testvm.stack) teststack
+if [ -f /usr/bin/openstack ]; then
+    openstack stack create -t $(readlink -e $PWD/testvm.stack) teststack
```

Comment:
whats the rationale behind the change? there used to be problems with openstackclient on juno iirc, which is why we're using the heat command.